package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "repository": "github:NUbots/NUsight2",
   "engines": {
-    "node": ">=10.0.0 <11.0.0",
+    "node": ">=10.0.0",
```

Comment:
This was just added 🤣 #260

Reply:
Well apparently it’s just 12 that’s a problem!

Reply:
If it needs updating let's move it to another PR.

Reply:
Reverted in https://github.com/NUbots/NUsight2/pull/262/commits/5cb557f38d2495c855fb05fded7bde091038eb82